doc/sources/installation/installation-linux-venvs.rst

```diff
@@ -0,0 +1,349 @@
+    sudo pip install --upgrade pip virtualenv setuptools
+
+    # Then create a virtualenv named "kivyinstall" by either:
+
+    # 1. using the default interpreter
+    virtualenv --no-site-packages kivyinstall
+
+    # or 2. using a specific interpreter
+    # (this will use the interpreter in /usr/bin/python2.7)
+    virtualenv --no-site-packages -p /usr/bin/python2.7 kivyinstall
+
+    # Enter the virtualenv
+    . kivyinstall/bin/activate
+
+    pip install numpy
+
+    pip install |cython_install|
+
+    # If you want to install pygame backend instead of sdl2
+    # you can install pygame using command below and enforce using
+    # export USE_SDL2=0. If kivy's setup can't find sdl2 libs it will
+    # automatically set this value to 0 then try to build using pygame.
+    pip install hg+http://bitbucket.org/pygame/pygame
+
+
+
+    # Install stable version of Kivy into the virtualenv
+    pip install kivy
+    # For the development version of Kivy, use the following command instead
+    pip install git+https://github.com/kivy/kivy.git@master
+
+
+Install additional Virtualenv packages
+--------------------------------------
+
+::
+
+    # Install development version of buildozer into the virtualenv
+    pip install git+https://github.com/kivy/buildozer.git@master
+
+    # Install development version of plyer into the virtualenv
+    pip install git+https://github.com/kivy/plyer.git@master
+
+    # Install a couple of dependencies for KivyCatalog
+    pip install -U pygments docutils
+
+
+.. _linux-run-app:
+
+
+Start from the Command Line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We ship some examples that are ready-to-run. However, these examples are packaged inside the package.
+This means you must first know where easy_install has installed your current kivy package,
+and then go to the examples directory::
+
+    $ python -c "import pkg_resources; print(pkg_resources.resource_filename('kivy', '../share/kivy-examples'))"
+
+And you should have a path similar to::
+
+    /usr/local/lib/python2.6/dist-packages/Kivy-1.0.4_beta-py2.6-linux-x86_64.egg/share/kivy-examples/
+
+Then you can go to the example directory, and run it::
+
+    # launch touchtracer
+    $ cd <path to kivy-examples>
+    $ cd demo/touchtracer
+    $ python main.py
+
+    # launch pictures
+    $ cd <path to kivy-examples>
+    $ cd demo/pictures
+    $ python main.py
+
+If you are familiar with Unix and symbolic links, you can create a link directly in your home directory
+for easier access. For example:
+
+#. Get the example path from the command line above
+#. Paste into your console::
+
+    $ ln -s <path to kivy-examples> ~/
+
+#. Then, you can access to kivy-examples directly in your home directory::
+
+    $ cd ~/kivy-examples
+
+If you wish to start your Kivy programs as scripts (by typing `./main.py`) or by double-clicking them,
+you will want to define the correct version of Python by linking to it. Something like::
+
+    $ sudo ln -s /usr/bin/python2.7 /usr/bin/kivy
```

Comment:
I'm hesitant with giving such advice, it's not really good practice, but i agree it can be a effective shortcut for new python users.

Reply:
This is just 1:1 copy+paste from the venv-section of the installation-linux.rst page. I have no problems removing or changing it if you know a better way...

Reply:
yeah, saw that after the fact, that's good for now, let's not mix reorganisation with too much content change.